content/getting-started/sections/updating-atom.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Updating Atom
+---
+### Updating Atom
+
+You will need to update Atom periodically for the latest improvements to the software. This section will explain how to update Atom on your system.
+
+#### Updating Atom in Windows
+
+Atom will check for updates automatically every 4 to 6 hours, but you can also update manually. To perform a manual update:
+
+1. Click on "Help" at the top of the Atom window.
+2. Click on "Check for Update".
```

Comment:
We should keep how we refer to the menu items consistent, see how the settings menu item is described in  https://flight-manual.atom.io/getting-started/sections/atom-basics/#settings-and-preferences for example (if you use the OS switcher at the top of the page, you can see how the description is different based on the OS).

So could be just:

> To perform a manual update, you can:
> 
> * Click on the `Help > Check for Update` menu item in the menu bar
> * Search for `Application: About` in the [Command Palette](https://flight-manual.atom.io/getting-started/sections/atom-basics/#command-palette) and click the `Check now` button
> 
> Atom will begin to update if there is an update available.